pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>io.imply</groupId>
   <artifactId>druid-example-extension</artifactId>
-  <version>0.10.0_1-SNAPSHOT</version>
+  <version>0.13.0-incubating_1-SNAPSHOT</version>
```

Comment:
The incubating here is noisy and, I think, unneeded (it’s not an apache release). I’d remove it.